pkg/apis/core/validation/validation.go

```diff
@@ -5613,6 +5616,22 @@ func validateServiceInternalTrafficFieldsValue(service *core.Service) field.Erro
 	return allErrs
 }
 
+// validateServiceTrafficDistribution validates the values for the
+// trafficDistribution field.
+func validateServiceTrafficDistribution(service *core.Service) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if service.Spec.TrafficDistribution == nil || *service.Spec.TrafficDistribution == "" {
```

Comment:
ok, it seems we are not going to use `Default`value and use nil as Default, I was thinking that was the best option too, and it is easy to implementn backwards compatibility and skew support

Reply:
Re-open.

I'm not sure unspecified makes sense in the new naming, and if it does make sense we should not equate "" with unspecified.  "" is a value.